### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by existentialist1888**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/RepoPage/RepoPage.js [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+    stars: '456',
+    links: 'Links',
+  },
+];
 
 const RepoPage = () => {
-  return <div>REPO PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
+      <div className="bx--row repo-page__r1">
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: Missing `<div className="bx--col-lg-16">` wrapper

File: src/content/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -1,7 +1,104 @@
...
+              <a href="/">Getting started</a>
+            </BreadcrumbItem>
+          </Breadcrumb>
+          <h1 className="landing-page__heading">
+            Design &amp; build with Carbon
+          </h1>
+        </div>
+      </div>
+      <div className="bx--row landing-page__r2">
+        <div className="bx--row landing-page__r2">
```
[the remark this answers is not in the document]
Comment (existentialist1888) [reply src/content/LandingPage/LandingPage.js]: The duplicate row has been deleted.

File: src/content/RepoPage/RepoPage.js (hunk 1 above)
Comment (existentialist1888) [reply src/content/RepoPage/RepoPage.js]: I have added the div around the table

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**